os/dbuild.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
To support Linux and Mac both, **_#!/usr/bin/env bash_** would be better.

Reply:
Or, below way is also working.

BUILD_OPT_LOWER=${BUILD_OPT,,}
to
BUILD_OPT_LOWER=$(echo "$BUILD_OPT" | tr '[:upper:]' '[:lower:]')

Because ${BUILD_OPT,,} is supported on bash 4.x or above.
Mac default bash version is 3.6. Please consider this.